src/routes/User.js

```diff
@@ -0,0 +1,82 @@
+import React, {useState, useEffect} from 'react';
+import axios from 'axios';
+import styled from 'styled-components';
+import Cookies from 'js-cookie';
+
+import Candidate from '../components/Candidate';
+
+function User({ history }) {
+  const candidatesUrl = 'http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:8080/candidates';
+  const voteUrl = `http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:8080/vote?id=`;
+  const [candidates, setCandidates] = useState([]);
+  const [rerenderTrigger, setRerenderTrigger] = useState(true);
+  let candidatesData = [];
+  let sortedCandidates = [];
+
+  useEffect(() => {
+    const getCandidate = async () => {
+      try {
+        const response = await axios.get(candidatesUrl);
+        candidatesData = response.data;
+      } catch (e) {
+        console.log('Candidates error : ', e);
+      }
+    }
+    
+    async function sortVotes() {
+      sortedCandidates = candidatesData
+        .sort((a, b) => {return b.voteCount - a.voteCount})
```

Comment:
return 문은 이렇게 하면 안 적어도 된답니당

```suggestion
        .sort((a, b) => b.voteCount - a.voteCount)
```